backend/codesirius/problems/models/execution_constraint.py

```
@@ -0,0 +1,80 @@
+                          One problem can have multiple execution constraints based
+                          on the language.
+        time_limit (int): The time limit for the execution of the problem.
+                         This is measured in seconds.
+        memory_limit (int): The memory limit for the execution of the problem.
+                            This is measured in megabytes.
+        language (Language): The language for which the execution constraints are
+                            applied.
+    """
+
+    problem = models.ForeignKey(
+        Problem, on_delete=models.CASCADE, related_name="execution_constraints"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)